rust-src/concordium_base/src/ffi_helpers/common.rs

```diff
@@ -6,3 +6,11 @@ extern "C" fn free_array_len(ptr: *mut u8, len: u64) {
         Vec::from_raw_parts(ptr, len as usize, len as usize);
     }
 }
+
+#[no_mangle]
+/// Free an array that was converted to a pointer from a vector.
```

Comment:
**Request change:** I find the comment and function name a bit misleading, since the function is for freeing a `Vec<u8>` specifically, not an array and not just any `Vec<T>`.

Reply:
At least it is not an array in Rust or C# terms, I guess it is an array in some other languages, like javascript.